apps/condo/domains/billing/schema/RegisterBillingReceiptsService.js

```diff
@@ -370,6 +372,15 @@ const RegisterBillingReceiptsService = new GQLCustomSchema('RegisterBillingRecei
                         partialErrors.push(new GQLError({ ...ERRORS.ADDRESS_EMPTY_VALUE, inputIndex: i }, context))
                         continue
                     }
+
+                    if (!normalizedAddress) {
+                        const [findedOrganizationProperty, score] = await findPropertyByOrganizationAndAddress(context, billingContext.organization.id, address)
+
+                        if (score > 80) {
```

Comment:
What the meaning of the `score` here? Please leave a comment here. It might be helpful for the future development process

Reply:
IMO that number can become a constant with clear name)

Reply:
The constant with description was added